backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -597,8 +593,11 @@ func backfillExperimentIDToRunTable(db *gorm.DB) error {
 
 // Returns the same error, if it's not "already exists" related.
 // Otherwise, return nil.
-func ignoreAlreadyExistError(err error) error {
-	if err != nil && strings.Contains(err.Error(), "exists") {
+func ignoreAlreadyExistError(driverName string, err error) error {
+	if driverName == "pgx" && err != nil && strings.Contains(err.Error(), "already exists") {
+		return nil
+	}
+	if driverName == "mysql" && err != nil && strings.Contains(err.Error(), "database exists") {
```

Comment:
We can define consts for each error message keyword `already exists` and `database exists` in a centralized place: https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/storage/db.go